src/export2mpb.jl

```diff
@@ -175,7 +175,7 @@ function prepare_mpbcalc!(
 
     # write `kvs` (if they are not nothing; we may not always want to give `kvs` explicitly,
     #              e.g. for berry phase calculations)
```

Comment:
```suggestion
    #              e.g. for berry phase calculations); note that we write them as "kvecs"
    #              rather than "kvs" for backwards compatibility with our .ctl files
```